### PR TITLE
cmake,CI: cmake-format cleanup

### PR DESCRIPTION
**Pull request by gxalpha**
### Description

cmake-format was replaced with gersemi in 19d3e30a3a1a4eeecbe41bed8768e15c71e9d4ce (#10438).
Comments for `cmake-format` are now without effect, and the `run-cmake-format` is unused (and would be broken due to the prior removal of the associated `run-cmake-format` script in the linked commit).
Both can be removed.

### Motivation and Context

Saw that these were still there when working on something else.

### How Has This Been Tested?

Ran `run-gersemi`, no changes.

### Types of changes

- Code cleanup (non-breaking change which makes code smaller or more readable)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: cmake/finders/FindFFmpeg.cmake [hunk 1]
```diff
@@ -93,10 +93,8 @@ The following cache variables may also be set:
 
 #]=======================================================================]
 
-# cmake-format: off
 # cmake-lint: disable=C0103
```
Comment (PatTheMav) [inline cmake/finders/FindFFmpeg.cmake:96]: Also remove all `cmake-lint` entries - that tool was part of `cmake-format` and `gersemi` takes care of some amount of linting itself.

**Review by PatTheMav — CHANGES_REQUESTED** [review]

File: cmake/finders/FindFFmpeg.cmake (hunk 1 above)
Comment (gxalpha) [reply cmake/finders/FindFFmpeg.cmake]: Removed.

Comment (PatTheMav) [reply cmake/finders/FindFFmpeg.cmake]: Were those the only remaining occurrences of `cmake-lint`?

Comment (gxalpha) [reply cmake/finders/FindFFmpeg.cmake]: Yes, according to git grep at least.

**Review by RytoEX — APPROVED** [review]

**Pull request merged**